style.css

```diff
@@ -91,3 +94,190 @@ header {
 .playfair {
   font-family: "Playfair Display", serif;
 }
+
+.bg1 {
+  background-image: url(images/icons/Project-1.svg);
+  background-position: top center;
+  height: 186px;
+  width: 320px;
+  top: 754px;
+}
```

Comment:
- [ ] I appreciate 😍😍your effort, but there are a few aspects that need alignment in order to match the suggested [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=0%3A1). 
- [ ] This will ensure that your project looks visually appealing and catches the attention of potential recruiters.🤔🤔
- [ ] To ensure that your project is mobile responsive within the range of `375px to 768px`, it is crucial to implement appropriate responsive design practices. This will guarantee that your project adapts and displays optimally on various mobile devices.
- [ ] Your design:
![screencapture-127-0-0-1-5500-index-html-2023-06-03-01_44_43](https://github.com/Saritalitu23/Microverse-Portfolio-Project-/assets/118761303/4d30f370-cf03-4cec-97c6-e21647036c97)